packages/react/.storybook/webpack.config.js

```diff
@@ -124,12 +107,52 @@ module.exports = ({ config, mode }) => {
     use: ['@svgr/webpack', 'url-loader'],
   });
 
+  const sassLoader = {
+    loader: 'sass-loader',
+    options: {
+      includePaths: [
+        path.resolve(__dirname, '..', 'node_modules'),
+        path.resolve(__dirname, '../../../', 'node_modules'),
+      ],
+      data: `
+        $feature-flags: (
+          ui-shell: true,
```

Comment:
Just wanted to let you know that `ui-shell` feature flag is no longer needed.

Reply:
Thanks @asudoh! I'll get rid of that 👍